rocketmq-docker/4.2.0/play-kubernetes.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Build base image
+docker build -t apache/rocketmq-base:4.2.0 --build-arg version=4.2.0 ./rocketmq-base
+
+# Build namesrv and broker
+docker build -t apache/rocketmq-namesrv:4.2.0-k8s ./kubernetes/rocketmq-namesrv
+docker build -t apache/rocketmq-broker:4.2.0-k8s ./kubernetes/rocketmq-broker
+
+# Then `docker tag` images, and `docker push` images to respective registry
+# For example:
+#docker tag apache/rocketmq-namesrv:4.2.0-k8s huanwei/rocketmq-namesrv:4.2.0-k8s
+#docker tag apache/rocketmq-broker:4.2.0-k8s huanwei/rocketmq-broker:4.2.0-k8s
+#docker push huanwei/rocketmq-namesrv:4.2.0-k8s
```

Comment:
Could you remove your private repository example, although it is be commented

Reply:
Sure.